src/QuickMarcEditor/utils.js

```diff
@@ -597,3 +597,9 @@ export const getContentSubfieldValue = (content) => {
     };
   }, {});
 };
+
+export const getLocationValue = ({ value }) => {
+  const matches = value?.match(/\$b\s([a-zA-Z]+\/*)+/) || [];
```

Comment:
I really like this regex, could you also use it in `validateLocationSubfield` in `utils.js` instead of `findLocationSubfieldValue` that I wrote?
You just need to remove `$b ` before comparison there